llm/server.go

```diff
@@ -156,11 +156,8 @@ func NewLlamaServer(gpus gpu.GpuInfoList, model string, ggml *GGML, adapters, pr
 		"--batch-size", fmt.Sprintf("%d", opts.NumBatch),
 		"--embedding",
 	}
-	if envconfig.Debug {
-		params = append(params, "--log-format", "json")
-	} else {
-		params = append(params, "--log-disable")
-	}
+
+	params = append(params, "--log-disable")
```

Comment:
In debug mode should we omit this?

Reply:
log disable specifies if it should log to a file, not disable logging entirely

```
        else if (arg == "--log-disable")
        {
            log_set_target(stdout);
            LOG_INFO("logging to file is disabled.", {});
        }
```